### ISSUE TITLE
[jest-dom/prefer-to-have-style] doesn't report when using `.toBe(variable)`

### ISSUE THREAD

**Issue by StyleShit** [testing-library/eslint-plugin-jest-dom#384]
- `eslint-plugin-jest-dom` version: 5.5.0
- `node` version: 18.20.3
- `npm` version: 10.2.3

**Relevant code or config:**

```JS
// eslintrc.js
{
	// ...
	rules: {
		'jest-dom/prefer-to-have-style': 'error',
	}
}
```

```JS
// component.test.js

expect(node.style.color).toBe('red'); // Works

const red = 'red';
expect(node.style.color).toBe(red); // Doesn't work
```

**What happened:**

I'm referencing a variable in the `toX()` methods, but the rule doesn't report.
I would've expected it to report. Is it intentional? Didn't find other issues mentioning this, but the selectors in the code don't seem to handle it.

**Reproduction repository:**

[StackBlitz](https://stackblitz.com/edit/vitejs-vite-3bjtgey2?file=eslint.config.js,src%2F__tests__%2Fa.test.js&terminal=dev)
